akka-http-core/src/test/scala/akka/http/scaladsl/ClientServerSpec.scala

```diff
@@ -447,7 +447,7 @@ class ClientServerSpec extends WordSpec with Matchers with BeforeAndAfterAll wit
         val serverInSub = serverIn.expectSubscription()
```

Comment:
Could you add a test that this PR directly makes pass @gosubpl ?
I.e. that we disabled it in config, but enabled in the routing layer, thus it should trigger properly